pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=56", "setuptools_scm"]
```

Comment:
`setuptools>=56` requires Python>=3.6, but requests-mock seems to still support Python 2.

Reply:
I guess it depends on what requests-mock wants to support. I forget why `setuptools>=56` was used. I can track it down if that would be helpful. Note that it is still possible to build wheels using late versions of Setuptools on Python 3 and then install that wheel to Python 2. If requests-mock only wishes to support installs from wheels or to require the build to be done with Python 3, then this constraint could be acceptable.